Add EXTRACT function to Rel8.Expr.Time

guaraqe opened the pull request:
This adds unsafe EXTRACT functions to Rel8.Expr.Time, using the low-level Opaleye functions. Should solve #236.

In principle a safe function can also be done with a GADT, but that would need more work.

guaraqe committed:
Add EXTRACT function to Rel8.Expr.Time

This adds unsafe EXTRACT function to Rel8.Expr.Time, using the low-level
Opaleye functions. Should solve #236.

ocharles commented:
I think I'd like to do this with some kind of GADT or something to track what is being extracted, rather than allowing any `DBType`.

idontgetoutmuch commented:
I need this so I think I will use @guaraqe's PR until something better comes along.

waiyaki committed:
Add forUpdate query locking helper

Expose a new Rel8.Query.Locking module and
re-export forUpdate from Rel8, wrapping Opaleye's
FOR UPDATE support for Rel8 queries.

guaraqe committed:
Merge pull request #1 from tloncorp/james/for-update

Add forUpdate query locking helper

guaraqe committed:
Merge branch 'circuithub:master' into master

guaraqe committed:
Revert "Merge branch 'circuithub:master' into master"

This reverts commit efe9165b4766cd773ef927a68ecb8ac306aa38ec, reversing
changes made to c4c98023df41f0c59aefdef7e60b68e5408da003.
